README.md

```diff
@@ -1,2 +1,127 @@
 # devices
 Servo-specific APIs to access various devices
+
+----------
+
+###How to use
```

Comment:
This README for the whole devices crate. We should specify that this part is for the `Bluetooth` part. (I know that the bt is the only one for now, but this can change)